nucleus/payara-modules/requesttracing-core/src/main/java/fish/payara/nucleus/requesttracing/store/LocalRequestTraceStore.java

```diff
@@ -63,7 +60,6 @@
         this.store = new BoundedTreeSet<>(0);
         this.maxStoreSize = store.size();
         this.strategy = strategy;
-        this.events = events;
```

Comment:
This does not have any effect, with it deleted it becomes apparent that the `events` field wasn't used at all so I removed it.